dist/src/main/arctic-bin/bin/ams.sh

```diff
@@ -18,28 +18,14 @@
 #
 
 CURRENT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
-AMORO_HOME="$( cd "$CURRENT_DIR/../" ; pwd -P )"
-export AMORO_HOME
 
-PRGDIR=$(dirname $0)
-PRGDIR=$(cd $PRGDIR;pwd)
-source ${PRGDIR}/config.sh
 
-
-CURRENT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
-APP_HOME=$CURRENT_DIR/../
+source ${CURRENT_DIR}/load-config.sh
 
 
-if [[ -d $JAVA_HOME ]]; then
-    JAVA_RUN=$JAVA_HOME/bin/java
-else
-    JAVA_RUN=java
-fi
-XMX=$XMX_CONFIG
-XMS=$XMS_CONFIG
 
-JAVA_OPTS="-server -Xloggc:$AMORO_HOME/logs/gc.log -XX:+IgnoreUnrecognizedVMOptions -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10M \
--Xms${XMS}m -Xmx${XMX}m \
+JAVA_OPTS="-server -Xloggc:$AMORO_LOG_DIR/gc.log -XX:+IgnoreUnrecognizedVMOptions -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10M \
+-Xms${JVM_XMS_CONFIG}m -Xmx${JVM_XMX_CONFIG}m \
```

Comment:
Should we default to using CMS as the garbage collector? In AMS with high availability mode, long stop the world pauses may cause Zookeeper heartbeat failures.

Reply:
I think this can be implemented in another PR.